### PR TITLE
only show sudo note with npm installation so no one is tempted to sud…

### PR DESCRIPTION
**Pull request by edwardfoyle**
…o curl scripts willy-nilly from the internet

Also updated one place where we were only showing the npm installation option

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/cli/start/install.md [hunk 1]
```diff
@@ -35,13 +37,10 @@ curl -sL https://aws-amplify.github.io/amplify-cli/install-win -o install.cmd &&
...
 
-> Because we're installing the Amplify CLI globally, you might need to run the command above with `sudo`.
-
-
 ### Pre-requisites for installation
 
-* [Install Node.js®](https://nodejs.org/en/download/) and [NPM](https://www.npmjs.com/get-npm) if they are not already on your machine.
-* Verify that you are running at least Node.js version 10.x and npm version 6.x or greater by running `node -v` and `npm -v` in a terminal/console window
+* If using the NPM installation option, you'll need to [install Node.js®](https://nodejs.org/en/download/) and [NPM](https://www.npmjs.com/get-npm) if they are not already on your machine.
+  * Verify that you are running at least Node.js version 10.x and npm version 6.x or greater by running `node -v` and `npm -v` in a terminal/console window
```
Comment (jhockett) [inline docs/cli/start/install.md:43]: nit: this looks like an accidental extra space

**Review by jhockett — APPROVED** [review]
LGTM, just one nit

File: docs/cli/start/install.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ The Amplify Command Line Interface (CLI) is a unified toolchain to create AWS cl
 npm install -g @aws-amplify/cli
 ```
 
+> Because we're installing the Amplify CLI globally, you might need to run the command above with `sudo`.
```
Comment (jamesonwilliams) [inline docs/cli/start/install.md:18]: Checkout the `<amplify-callout>` tag. Might be a better candidate for this.

File: docs/cli/start/install.md [hunk 3]
```diff
@@ -35,13 +37,10 @@ curl -sL https://aws-amplify.github.io/amplify-cli/install-win -o install.cmd &&
...
 </amplify-block-switcher>
 
-> Because we're installing the Amplify CLI globally, you might need to run the command above with `sudo`.
-
-
 ### Pre-requisites for installation
 
-* [Install Node.js®](https://nodejs.org/en/download/) and [NPM](https://www.npmjs.com/get-npm) if they are not already on your machine.
-* Verify that you are running at least Node.js version 10.x and npm version 6.x or greater by running `node -v` and `npm -v` in a terminal/console window
+* If using the NPM installation option, you'll need to [install Node.js®](https://nodejs.org/en/download/) and [NPM](https://www.npmjs.com/get-npm) if they are not already on your machine.
```
Comment (jamesonwilliams) [inline docs/cli/start/install.md:42]: What's the other option?

Comment (ericclemmons) [reply docs/cli/start/install.md:42]: I think it's referring to the cURL option above (that's hidden in the diff):

```shell
curl -sL https://aws-amplify.github.io/amplify-cli/install | bash && $SHELL
```

**Pull request closed without merging**